snap/local/helper-go/common.go

```diff
@@ -38,6 +39,7 @@ var (
 	securitySetupServices = []string{
 		securitySecretStoreSetup,
 		securityBootstrapperConsul,
+		coreCommonConfigBootstrapper,
```

Comment:
`coreCommonConfigBootstrapper` is not a security setup service. If kept in this group, disabling security will also disable the config bootstrapper.

You may want to instead add another group called setupServices and add that group to all needed places, e.g. the `allServices()` function and the autostart unsetting in `configure()` function.